app/controllers/posts_controller.rb

```diff
@@ -35,6 +36,10 @@ def update
   end
 
   def destroy
+    @post = Post.find(params[:id])
```

Comment:
We should be able to add an `if` statement here, to determine the details of the:

* post's user
* current_user

and can then compare them.  If the `current_user` isn't the `post.user`, I think we should show an error message.